pkg/k8s/watchers/cilium_endpoint_batch_subscriber.go

```diff
@@ -0,0 +1,159 @@
+//  Copyright 2021 Authors of Cilium
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package watchers
+
+import (
+	"github.com/cilium/cilium/pkg/k8s"
+	cilium_v2a1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	k8sUtils "github.com/cilium/cilium/pkg/k8s/utils"
+
+	"github.com/sirupsen/logrus"
+)
+
+type CEBSubcriber struct {
```

Comment:
Nit: consider only exporting objects / methods only if they're referenced outside the package. This seems like it should just be `cebSubscriber`.

Reply:
SG Done.